packages/overlay/src/vaadin-overlay.js

```diff
@@ -580,15 +574,15 @@ class Overlay extends ThemableMixin(DirMixin(ControllerMixin(PolymerElement))) {
     }
   }
 
-  /** @protected */
+  /** @private */
   _attachOverlay() {
     this._placeholder = document.createComment('vaadin-overlay-placeholder');
     this.parentNode.insertBefore(this._placeholder, this);
     document.body.appendChild(this);
     this.bringToFront();
   }
 
-  /** @protected */
+  /** @private */
```

Comment:
`_finishOpening`, `_finishClosing`, `_flushAnimation` should be marked as protected in JSDoc to align with the TS declarations.

Reply:
Marked `_flushAnimation` as protected for now as it's used by other components. 
Other methods are now marked as private, we don't need to override them.